### PR TITLE
[Reviewing][Filled with legal terms]Legal/Copyright

### PR DESCRIPTION
**Pull request by axmmisaka**
### Description

Chinese translation of Legal/Copyright

### Status
Partially done and they are waiting for review.
The rest part is WIP.
The translator will take part in some personal issues day later, so this PR will probably last for very long time.
Feel free to fork this branch and close this PR if you want to, since I may disappear for a long time (or not).

### Checklist 
Translations of a lot legal terms, such as
- [x] believes in good faith (a legal translation)
- [x] under penalty of perjury (do have a not-that-good translation)
- [x] Counter-Notices (what is this?)
and
- [x] Whether ppy's address should be translated into Chinese (if anyone wanna really post one they'll probably refer to the English version... That being said the Russian version is not translated.)

### CODE REVIEW

**Comment by benpigchu** [comment]
【 先提一下：非英文版本的法律文档需注明“翻译版不具有法律效力，以英文原文为准” 】

File: wiki/Legal/Copyright/zh.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+若指定代理人收到反对通知书，osu!会向原始举报者发送一份反对通知书的副本以声明被移除的内容可能在10个工作日内被恢复或解除访问阻止。
+
+除非版权持有者采取行动以要求反对内容提供者、会员或用户之法院命令，被移除的内容在osu!独立自由裁量权下在收到反对通知书10到14个或更长的工作日后可能被恢复或解除访问阻止。
+
+## 指定代理人联系方式
+
+**电子邮件地址**：[copyright@ppy.sh](mailto:copyright@ppy.sh) (保证24小时内回复)
+
+**邮寄地址**：
```
Comment (kj415j45) [inline wiki/Legal/Copyright/zh.md]: 直接使用原文

File: wiki/Legal/Copyright/zh.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+osu!的指定接受侵权声明通知的代理人（“指定代理人”）的联系信息列在此政策最后，并记录在美利坚合众国版权办公室的文件中 (http://www.copyright.gov/onlinesp/list/index.html)。
+
+osu!十分严肃地对待版权和其它知识产权。
+
+
+osu!的政策将会（1）迅速阻止访问或移除被诚意的认为（believes in good faith）可能包含侵犯了第三方版权的材料并（2）移除持续侵权者并停止对他们的服务。
+
+
+
+#举报版权侵犯
```
Comment (kj415j45) [inline wiki/Legal/Copyright/zh.md]: 空格

**Review by kj415j45 — CHANGES_REQUESTED** [review]
空格

**Review by kj415j45 — COMMENTED** [review]
空格。。。

**Pull request merged**